src/index.ts

```diff
@@ -1,7 +1,7 @@
-// Export constructs here
-export * as aws_cur from './aws-cur';
+// ~~ Generated by projen. To modify, edit .projenrc.ts and run "npx projen".
 export * as aws_codeartifact from './aws-codeartifact';
+export * as aws_cur from './aws-cur';
 export * as aws_ec2 from './aws-ec2';
 export * as aws_elasticache from './aws-elasticache';
 export * as aws_fsx from './aws-fsx';
-export * as aws_redshiftserverless from './aws-redshiftserverless';
+export * as aws_redshiftserverless from './aws-redshiftserverless';
```

Comment:
quibble: we can still add a newline at the end here

Reply:
Looks nice and are add elsewhere, nothing else. Hence marking this [quibble](https://conventionalcomments.org/). Absolutely non blocking.